doc/model.md

```diff
@@ -1,162 +1,31 @@
-
-Représente une date.
-
-Formats supportés :
-* "yyyyMMdd"
-* "yyyy"
-* "dd/MM/yyyy"
-
-Format normalisé : "yyyyMMdd"
-
-### Geometry (org.locationtech.jts.geom.Geometry)
-
-Remarque :
-* Les types dérivés Point, LineString, Polygon, MultiPoint, MultiLineString, MultiPolygon, GeometryCollection
-permettent de restreindre les types de géométrie autorisées.
-* Des conversions automatiques sont effectuées quand elles sont possibles (MultiPoint avec un seul Point => Point).
-* Le validateur signale par des erreurs les conversions impossibles (elles peuvent bloquer l'intégration dans des systèmes du type PostGIS)
-
-### Path (java.io.File)
-
-Représente un chemin vers un fichier contenu dans le répertoire du document.
-
-Contrôle spécifique : Présence du fichier dans le répertoire du document.
-
-### Url (java.net.URL)
-
-Représente un lien hypertexte.
-
-Contrôle spéficique : Validité de l'URL (pas de tentative de téléchargement).
-
-
-# Principe de fonctionnement
-
-1) Chargement du modèle de document et préparation d'un dossier de validation
-
-Le validateur prépare un répertoire de travail pour la validation où il écrira :
-* Le rapport de validation
-* Le répertoire DATA contenant les fichiers normalisés
-
-2) Enumération des fichiers
-
-3) Recherche des modèles de fichier correspondant aux fichiers
-
-Remarque : Le validateur signale alors les fichiers sans correspondance avec le modèle et les fichiers non trouvés
-
-4) Validation de chaque modèle de fichier
-
-Pour chaque modèle de fichier, le validateur valide les fichiers associés.
-
-Remarques :
-
-* Dans le cas des tables, le validateur valide la structure (FeatureType) et les attributs (AttributeType)
-* La validation des types d'attribut repose sur la possibilité de convertir une valeur dans le type décrit. Pour résumer : false, f, 0 sont des booléens, tutu n'est pas un booléen.
-
-5) Ecriture du rapport d'erreur
-
-Le validateur écrit un rapport dans le dossier
-
-6) Normalisation des données
-
-Le validateur génère un dossier DATA "à plat" contenant tous les fichiers validés et normalisés.
-
-# Extensibilité
-
-Le validateur permet l'ajout qui plugin qui vont exécuter des tâches à différentes étapes de la validation :
-
-* Avant la mise en correspondance des fichiers et du modèle (ex : modification d'extension)
-* Avant la validation (ex : détection de l'encodage des fichiers à partir des métadonnées)
-* Après la validation (ex : receuil de métadonnées sur les données validée, contrôle supplémentaires, etc.)
-
-Remarque : La normalisation des données est en pratique le premier post-traitement.
+* Les formats suivants sont supportés :
+  * Shapefile
+  * Mapinfo
+  * CSV
+  * GML (Simple Feature, i.e. une seule table dans le GML)
+* Le champ géométrique doit être décrit sous le nom "WKT"
+
+Remarque : Ce dernier point est lié au comportement par défaut de ogr2ogr lors de la conversion en CSV. Une prochaine version du validateur apportera de la souplesse sur ce point.

```

Comment:
Is there an open issue concerning the name of the geometry field ?
Maybe we can be specific on which target version (4.1 ?) will be more permissive.

Reply:
Issue added with milestone 4.1 : https://github.com/IGNF/validator/issues/192